src/ai/backend/client/cli/session/lifecycle.py

```diff
@@ -1265,11 +1265,8 @@ def events(session_name_or_id, owner_access_key, scope):
 
         async def _run_events():
             async with AsyncSession() as session:
-                try:
-                    session_id = uuid.UUID(session_name_or_id)
-                    compute_session = session.ComputeSession.from_session_id(session_id)
-                except ValueError:
-                    compute_session = session.ComputeSession(session_name_or_id, owner_access_key)
```

Comment:
The intention of the code appears to be to raise a `ValueError` if `session_name_or_id` is a str that is not a `UUID`, and to call `from_session_id` if it is a session ID. Otherwise, the default constructor is intended to be called.

However, when `from_session_id` is called, the `name` field is not populated, which causes the code to not function correctly.

Additionally, it seems to be a mistake that `owner_access_key` is not set when `session_id` is of UUID type.